draft-ietf-rats-corim.md

```diff
@@ -1066,6 +1066,182 @@ The following describes each member of the `concise-bom-tag` map.
+
+The results of the evidence collection are protocol specific data and transcripts which can be processed by the verifier.
+
+### Cryptographic validation of Evidence
+
+If the authenticity of Evidence is secured by a cryptographic mechanism such as a signature, the first step in the Evidence Appraisal is to perform cryptographic validation of the Evidence.
+
+The exact cryptographic signature validation mechanics depend on the specific Evidence collection protocol.
+
+For example:
+In DICE, a proof of liveness is performed on the final key in the certificate chain. If this passes then a suitable certification path anchored on a trusted root certificate is looked up -- e.g., based on linking information obtained from the DeviceID certificate (see Section 9.2.1 of [DICE Layering Architecture](https://trustedcomputinggroup.org/wp-content/uploads/DICE-Layering-Architecture-r19_pub.pdf))-- in the Appraisal Context.  If found, then usual X.509 certificate validation is performed.
+In PSA, the verification public key is looked up in the appraisal context using the `euid` claim found in the PSA claims-set (see [Section 4.2.1 of I-D.tschofenig-rats-psa-token](https://www.ietf.org/archive/id/draft-tschofenig-rats-psa-token-10.html#name-instance-id).  If found, COSE Sign1 verification is performed accordingly.
+
+Independent of the specific method, the cryptographic integrity of Evidence MUST be successfully verified.
+
+> A CoRIM profile MUST describe:
+>
+> * How cryptographic verification key material is conveyed (e.g., using Attestation Keys triples, or CoTS tags)
+> * How key material is associated with the Attesting Environment
+> * How the Attesting Environment is identified in Evidence
+
+### The Accepted Claims Set
+
+At the end of the Evidence collection process evidence has been converted into a format suitable for appraisal. Verifiers are not required to use this as their internal state, but for the purposes of this document a sample verifier is discussed which uses this format.
+
+The accepted claims set will be matched against CoMID reference values, so this document describes an evidence structure which can be easily matched against these reference values. Each set of evidence contains an environment map providing a namespace, and a measurement-values-map containing one or more entries.
+
+Each entry in the measurement-values-map is a separate piece of evidence describing the environment named by the environment-map. An attestor can provide multiple environment-maps each containing a single values measurement-values map, a single environment-map containing multiple entries in its measurement-values-map, or a combination of these approaches.
+
+If evidence from different sources has the same environment-map then the measurement-values-maps are merged. If both measurement-value-maps being merged contain the same key then the values associated with that key MUST be binary identical. 
```

Comment:
suggest "... the measurement-values-maps are..." to "... the measurements (e.g., `measurement-values-map`) are merged.
'key' is ambiguous - suggest 'code point' 
suggest adding a simple example - "(e.g., if the 'name' code point has the value "my environment" and 'name' is found in multiple sets of measurements being merged, then each value must be "my environment" or the merge will fail."